pkg/kubelet/lifecycle/handlers.go

```diff
@@ -47,6 +49,11 @@ const (
 
 	AppArmorNotAdmittedReason          = "AppArmor"
 	PodLevelResourcesNotAdmittedReason = "PodLevelResourcesNotSupported"
+
+	// Reasons for pod features admission failure
```

Comment:
Any new reasons should be added to https://github.com/kubernetes/kubernetes/blob/d8a53105dda26cb46aa84e8b6d4eb2c7a1421215/pkg/kubelet/kubelet.go#L242